routes/words.py

```diff
@@ -31,30 +30,94 @@
 
 from settings import changedlocale
 
+from tokenizer import TOK
+
 from reynir.bindb import BIN_Db
+from reynir.bintokenizer import tokenize
 
 from db import SessionContext, desc
-from db.models import Article, Root, Location, ArticleTopic, Topic
+from db.models import Article, Word, Root
 from db.queries import WordFrequencyQuery
 
 
 @routes.route("/words")
 def words():
     """ Handler for word frequency page. """
-    return render_template("words.html", title="Orð")
+    return render_template("words/freq.html", title="Orð")
+
+
+@routes.route("/words_trends")
+def words_trends():
+    """ Handler for word trends page. """
+    return render_template("words/trends.html", title="Orð")
+
+
+# Word categories permitted in word frequency search
+_VALID_WCATS = frozenset(
+    ("kk", "kvk", "hk", "lo", "so", "person_kk", "person_kvk", "entity", "??")
```

Comment:
Add "person_hk" - I think that is a possibility in the current code and will be seen more in the future